themes/finna2/templates/organisationinfo/elements/widget.phtml

```diff
@@ -0,0 +1,34 @@
+<!-- START of: finna - organisationinfo/elements/widget.phtml -->
+<div class="location-details info-element">
+  <div class="<?=$showDetails ? 'info-col ' : ''?>info">
+    <div class="js-location-dropdown dropdown">
+      <button type="button" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" data-id="">
```

Comment:
Lisätään tähän aria-live. Pitää varmaakin lisätä aria-live js:n kautta, jotta vältetään tilanne, että ruudunlukija sanoo sivun latauksen yhteydessä palvelupisteen. Varmaan joutuu lisää span-luokan sisälle, koska aria-liveä ei ehkä kannatta laittaa button-elementtiin.
`      <button type="button" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" data-id="">
        <span aria-atomic="true" aria-live="polite">
          <span class="sr-only"><?=$this->transEsc('organisation_info_selected_service_point')?></span>
          <span class="js-selected"><?=$this->transEsc($locationName ?: 'choose_from_list')?></span>
          <?=$this->icon('dropdown-open') ?>
        </span>
      </button>`

Reply:
Lisätty.